src/transmissionSettings/SettingsCheckbox.tsx

```diff
@@ -0,0 +1,44 @@
+import { ListItem, ListItemIcon, ListItemText } from '@mui/material'
```

Comment:
Not really sure what these components are going to end up looking like, I just know I want them to be opinionated. There's like 30 different fields that should all behave vaguely similarly.